onnxruntime/contrib_ops/cpu/bert/gqa_attention_base.h

```diff
@@ -208,7 +208,7 @@ class GQAAttentionBase : public AttentionBase {
                                bool packed_qkv,                     // whether Q, K, V are packed
                                ThreadPool* tp) const {
     const bool is_prompt = sequence_length != 1;
-    const int packed_batch_stride = packed_qkv ? (num_heads_ + 2 * kv_num_heads_) * sequence_length * head_size : 0;
+    const int64_t packed_batch_stride = packed_qkv ? SafeInt<int64_t>(num_heads_ + 2 * kv_num_heads_) * sequence_length * head_size : 0;
```

Comment:
Why do the types in these two lines differ?

https://github.com/microsoft/onnxruntime/blob/b65ad66812ee12a5235c197f465db96f25561e0f/onnxruntime/contrib_ops/cpu/bert/gqa_attention_base.h#L101
https://github.com/microsoft/onnxruntime/blob/b65ad66812ee12a5235c197f465db96f25561e0f/onnxruntime/contrib_ops/cpu/bert/gqa_attention_base.h#L211

Reply:
ptrdiff_t are 32-bit on the 32-bit platform and 64-bit on the 64-bit platform. 32 bit platform will not able to run large seq length  anyway.